.github/workflows/causal_lm_cpp.yml

```diff
@@ -703,12 +703,13 @@ jobs:
           source ./ov/setupvars.sh
           python -m pip install ./thirdparty/openvino_tokenizers/[transformers] --pre --extra-index-url https://storage.openvinotoolkit.org/simple/wheels/nightly
           python -m pip install --upgrade-strategy eager -r ./samples/requirements.txt --pre --extra-index-url https://storage.openvinotoolkit.org/simple/wheels/nightly
-          python ./samples/cpp/visual_language_chat/export_MiniCPM-V-2_6.py ./miniCPM-V-2_6/
+          python -m pip install -U "optimum<1.23" --no-dependencies
```

Comment:
should it be moved to `samples/requirements.txt` ?

Reply:
It's a conflicting thing. The only possible thing is to add it to README.md. But I want to see if it fixes by itself before the release